MAUI/DataForm/ManualDataFormItem.md

```diff
@@ -0,0 +1,228 @@
+        <ContentPage.BindingContext>
+            <local:DataFormViewModel />
+        </ContentPage.BindingContext>
+
+        <ContentPage.Content>
+            <dataForm:SfDataForm x:Name="dataForm" DataObject="{Binding Details}"  AutoGenerateItems="false"> 
+                <dataForm:SfDataForm.Items> 
+                    <dataForm:DataFormTextItem FieldName="Name" /> 
+                    <dataForm:DataFormTextItem FieldName="Password" /> 
+                    <dataForm:DataFormTextItem FieldName="Address" /> 
+                    <dataForm:DataFormAutoCompleteItem FieldName="Country"  ItemsSource = "{Binding CountryNames}"/> 
+                </dataForm:SfDataForm.Items> 
+            </dataForm:SfDataForm>
+        </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+    DataFormViewModel contactInfoViewModel = new DataFormViewModel(); 
+    dataForm.DataObject = contactInfoViewModel.ContactDetails; 
+    ObservableCollection<DataFormViewItem> items = new ObservableCollection<DataFormViewItem>(); 
+    items.Add(new DataFormTextItem() { FieldName = "Name" }); 
+    items.Add(new DataFormPasswordItem() { FieldName = "Password" }); 
+    items.Add(new DataFormMultilineItem() { FieldName = "Address" }); 
+    items.Add(new DataFormAutoCompleteItem() { FieldName = "Country", ItemsSource = contactInfoViewModel.CountryNames }); 
+    dataForm.AutoGenerateItems = false; 
+    dataForm.Items = items;
+
+{% endhighlight}   
+{% endtabs %} 
+
+{% endhighlight %}
+{% highlight C# %}
+
+    public class DataFormViewModel
+    {
+        public DataFormViewModel()
+        {
+            this.ContactDetails = new ContactInfo();
+            this.CountryNames = new List<string>
+            {
+                "United States",
+                "United Kingdom",
+                "France",
+                "Belgium",
+                "Germany"
+            };
+        }
+
+        public ContactInfo ContactDetails { get; set; }
+
+        public IList<string> CountryNames { get; set; }
+
+    }
+
+    private IList GetItemSource(string sourceName) 
+    { 
+        var list = new List<string>(); 
+        if (sourceName == "Countries") 
+        { 
+            list.Add("India");
+            list.Add("Japan");
+            list.Add("China");
+            list.Add("USA");
+            list.Add("America");
+            list.Add("UK");
+        } 
+        return list; 
+    }
+{% endhighlight %}
+{% endtabs %}
+
+{% endhighlight %}
+{% highlight C# %}
+
+    public class ContactInfo
+    {
+        public string Name { get; set; }
+
+        public string Password { get; set; }
+
+        public string Address { get; set; }
+
+        public string Country { get; set; }
+
+        public string Email { get; set; }
+    }
+
+{% endhighlight %}
+
+### Adding DataForm group manually
+Support has been provided to dynamically add the dataform items to collections using the [Items]`Link` property of `SfDataForm`.
```

Comment:
You can also generate [DataFormGroupItem] manually using the [Items] property of `SfDataForm`

Reply:
content updated